### PR TITLE
FAQ - New exposure check intervals (< 24h)

### PR DESCRIPTION
**Pull request by heinezen**
Resolves https://github.com/corona-warn-app/cwa-website/issues/628

### CODE REVIEW

**Comment by MikeMcC399** [comment]
https://www.coronawarn.app/en/faq/#multiple_exposure_checks says;

"When the app downloads the current diagnosis keys from the server, a check of the exposure log is performed. During that process, a separate check is performed for each of the last 14 days, for which diagnosis keys are available. It is therefore not an error that multiple, seemingly simultaneous entries are visible in the exposure check history. You can further click on the details of each check to see that different amounts of keys were provided for each of them."

There are no longer "multiple, seemingly simultaneous entries visible in the exposure check history" and this is not expected when CWA has been updated to the current version of the app >= 1.6.x.

Starting with CWA 1.7.x the app checks a minimum of once a day when the device only has connectivity to a mobile data network **and** every 4 hours (in other words 6 times a day) with WiFi connectivity. It does **not** cause 14 entries with more or less the same timestamp in the exposure log any more.

Could we get the entry https://www.coronawarn.app/en/faq/#multiple_exposure_checks in the FAQs also updated accordingly?

**Review by loki-cui — CHANGES_REQUESTED** [review]
typo in german translation -> last word = "abgefragr" instead of "abgefragt"

File: src/data/faq_de.json [hunk 1]
```diff
@@ -517,9 +517,7 @@
...
-                            "Dass nur etwa alle 24 Stunden aktualisiert wird, liegt zum einen an Beschränkungen der Schnittstelle und gewährleistet zum anderen die Konsistenz zwischen den Corona-Warn-App-Versionen für iOS und Android.",
-                            "Eine ausführliche technische Begründung dafür finden Sie in englischer Sprache in Github unter <a href='https://github.com/corona-warn-app/cwa-backlog/issues/2#issuecomment-647740143' target='_blank' rel='noopener noreferrer'>https://github.com/corona-warn-app/cwa-backlog/issues/2#issuecomment-647740143</a>.",
-                            "Das Thema ist aufgenommen und wird so bald wie möglich von der Entwicklung bearbeitet."
+                            "Seit Version 1.7 der Corona-Warn-App wird der Risikostatus bis zu 6 mal täglich aktualisiert, wenn eine WLAN-Verbindung vorhanden ist. Die wiederholte Abfrage wird automatisch durchgeführt. Bei einer mobiilen Internetverbindung wird der Risikostatus aufgrund der Zero-Rating Verpflichtungen der Provider lediglich einmal täglich abgefragr."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:520]: mobiilen => mobilen

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Bitte Schreibfehler korrigieren!

**Comment by MikeMcC399** [comment]
Please also update the current app version numbers on the main page https://www.coronawarn.app/en/, where it still says:

"1. Download the current version of the app (iOS: 1.6.1 or Android: 1.6.1)"

whereas the changed files in this PR already reference version 1.7.

**Comment by dsarkar** [comment]
@MikeMcC399, @loki-cui 

corrected-->

mobile
abgefragt.

**Review by dsarkar — APPROVED** [review]

**Review by loki-cui — APPROVED** [review]

**Pull request merged**